template/pyproject.toml.jinja

```diff
@@ -42,14 +41,6 @@ classifiers = [
     {% elif license == "Apache-2.0" %}
     "License :: OSI Approved :: Apache Software License",
     {% endif -%}
-    "Natural Language :: English",
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3 :: Only",
```

Comment:
If we remove the `requires-python` then the version-specific classifiers at least should go. I expanded to remove the others, as again this seems like a lot of assumed project intent that isn't tied to any user queries.

Reply:
I would at minimum leave the 3 Only classifier. PyPI uses these classifiers for filtering and search.